Core/views.py

```diff
@@ -116,8 +119,44 @@ class RegistrationView(FormView):
     def post(self, request, *args, **kwargs):
         alert = ''
         form = RegistrationForm(request.POST)
+
         if form.is_valid():
+            event_receiver_email = form.cleaned_data['email']
             form.save()
+
+            allowed = False
+
+            try:
+                allowed = (Email_Content.objects.get(event=self.event)).mail_allowed
+            except:
```

Comment:
Please don't use bare `except` clauses, catch specific errors like
```suggestion
            except Email_Content.DoesNotExist:
```